web_pwa_oca/templates/assets.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="web_layout_pwa" name="Web layout PWA" inherit_id="web.layout">
+        <xpath expr="//meta[@name='viewport']" position="after">
+            <!-- Add link rel manifest -->
+            <link rel="manifest" href="/web_pwa_oca/manifest.json"/>
+            <!-- Add iOS meta tags and icons -->
+            <meta name="apple-mobile-web-app-capable" content="yes"/>
+            <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
+            <meta name="apple-mobile-web-app-title" content="Odoo PWA"/>
+            <link rel="apple-touch-icon" href="/web_pwa_oca/static/img/icons/icon-152x152.png"/>
+            <!-- Add meta theme-color -->
+            <meta name="theme-color" content="#2E69B5" />
+        </xpath>
+    </template>
+    <template id="assets_backend" name="web service worker assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/web_pwa_oca/static/src/js/pwa_manager.js"></script>
+            <script type="text/javascript" src="/web_pwa_oca/static/src/js/pwa_install.js"></script>
+            <script type="text/javascript" src="/web_pwa_oca/static/src/js/webclient.js"></script>
+        </xpath>
+    </template>
+    <template id="manifest" name="PWA manifest">
+        {
+          "name": "<t t-esc="pwa_name"/>",
```

Comment:
As told above, wouldn't it be better to just have one controller that returns a JSON object? That'd be much easier to implement  and extend.

Just think what would happen if anybody sets this parameter to `My very "cool" website`. Now think all the possible corner cases... :shrug: